internal/client/clients.go

```diff
@@ -80,110 +103,99 @@ func (c *Client) newConditionOrcClient() error {
 		return ErrNilConfig
 	}
 
-	var coClient *conditionOrcapi.Client
 	var err error
-
-	if c.cfg.FdbCfg.DisableOAuth {
-		coClient, err = conditionOrcapi.NewClient(c.cfg.FdbCfg.Endpoint)
+	var client *http.Client
+	var secret string
+	if c.cfg.CoCfg.DisableOAuth {
+		secret = "dummy"
+
+		logHookFunc := func(l retryablehttp.Logger, r *http.Response) {
```

Comment:
I agree. The log messages in there are specific to each client. I dont know of a trick to make that not the case, or maybe I shouldnt have to worry about what client it comes from, as it will error up the chain and that will inform us what client broke.